birdhouse/components/scheduler/deploy_data_job.env

```diff
@@ -33,7 +33,7 @@ fi
 # Location for local cache of git clone to save bandwidth and time from always
 # re-cloning from scratch.
 if [ -z "$DEPLOY_DATA_JOB_CHECKOUT_CACHE" ]; then
-    DEPLOY_DATA_JOB_CHECKOUT_CACHE="/data/deploy_data_cache/$DEPLOY_DATA_JOB_JOB_NAME"
+    DEPLOY_DATA_JOB_CHECKOUT_CACHE="${DATA_PERSIST_ROOT:-/data}/deploy_data_cache/${DEPLOY_DATA_JOB_JOB_NAME}"
```

Comment:
For this one, I'm not too sure what will happen since it is not a template file that would be processed by `pavics-compose`. Will it actually find the value from `default.env`? If so, indeed doesn't need the default again.

Reply:
This `deploy_data_job.env ` is sourced in `env.local` https://github.com/bird-house/birdhouse-deploy/blob/02d1e9db0cfc0ca0c2ca382e379228a48d4ca182/birdhouse/env.local.example#L180 and `env.local` should always be read after `default.env` so should not need to repeat the default again.